rules/terraform_locals_order.go

```diff
@@ -0,0 +1,114 @@
+package rules
+
+import (
+	"github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-terraform/project"
+	"sort"
+)
+
+// TerraformLocalsOrderRule checks whether all arguments inside a `locals` block are sortedByAlphabetOrder in alphabet order
+type TerraformLocalsOrderRule struct {
```

Comment:
Please also change the file name and the struct name to `terraform_ordered_locals`(`TerraformOrderedLocalsRule`) to match the rule name.